app/src/main/java/guessme/ppc2017/unnoba/edu/ar/gameone/MathOperation.java

```diff
@@ -1,30 +1,93 @@
 package guessme.ppc2017.unnoba.edu.ar.gameone;
 
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
+import java.util.Random;
 
 /**
  * Created by anton on 9/4/2017.
  */
 
 public class MathOperation {
```

Comment:
I'm pretty sure that this class could be improved to look itself nicer, I don't know.. maybe using arrays. I don't like the method calculateResultOfOperation(...) but at the moment I don't have a solid idea to replace it for something better. If you can't find a better solution, it's ok leaving this one just like it is